chalicelib_fourfront/checks/helpers/wfr_utils.py

```diff
@@ -189,11 +189,13 @@
     # OFFICIAL - 1 STEP
     '2-stage Repli-seq': ['RepliSeq_Pipeline_v13.1_step1',
                           'RepliSeq_Pipeline_v14_step1',
-                          'RepliSeq_Pipeline_v16_step1'],
+                          'RepliSeq_Pipeline_v16_step1',
+                          'RepliSeq_Pipeline_v16.1_step1'],
     # OFFICIAL - 1 STEP
     'Multi-stage Repli-seq': ['RepliSeq_Pipeline_v13.1_step1',
                               'RepliSeq_Pipeline_v14_step1',
-                              'RepliSeq_Pipeline_v16_step1'],
+                              'RepliSeq_Pipeline_v16_step1',
+                              'RepliSeq_Pipeline_v16.1_step1'],
     # Preliminary - Released to network
     'NAD-seq':       ['RepliSeq_Pipeline_v13.1_step1', 'RepliSeq_Pipeline_v14_step1', 'RepliSeq_Pipeline_v16_step1'],
```

Comment:
Should v16.1 be added here and in TSA-seq section above?

Reply:
Good point!